src/Translation.php

```diff
@@ -156,39 +156,39 @@ public static function for_incoming(array $incoming_data)
     /**
      * @inheritdoc
      */
-    public function offsetExists($offset)
+    public function offsetExists($offset): bool
     {
         return $this->has_value($offset);
     }
 
     /**
      * @inheritdoc
      */
-    public function offsetGet($offset)
+    public function offsetGet($offset): mixed
```

Comment:
:thinking: 

This type is available only in PHP 8.

Reply:
Thank you and fixed here: d99d2ee6cbd7f82fa3e272f5a93e625aaca7d641